src/Features/Blockcore.Features.Consensus/FullNodeBuilderConsensusExtension.cs

```diff
@@ -28,7 +28,8 @@ public static IFullNodeBuilder UsePowConsensus(this IFullNodeBuilder fullNodeBui
                     .AddFeature<PowConsensusFeature>()
                     .FeatureServices(services =>
                     {
-                        AddCoindbImplementation(services, coindbType);
+                        fullNodeBuilder.PersistenceProviderManager.RequirePersistence<ConsensusFeature>(services);
```

Comment:
is require the correct terminology here?
its more like .Configure() or .Register()

Reply:
it's opinable, I'm fine to change it to something different, anyway in that place the feature is requiring a persistence implementation, the fact that the persistenceprovidermanager register it as service is an implementation detail, here you could think PersistenceProviderManager as a IServiceProvider so maybe just "GetPersistence<ConsensusFeature>" could be fine (like IServiceProvider .GetService)

Reply:
ah yeah I realize its a manager of providers so not what I was thinking well then leave the name as is.